test/test_load_simulation.jl

```diff
@@ -0,0 +1,91 @@
+include("../Simulations_preload.jl");
+
+branches = Dict{Symbol, DeviceModel}()
+services = Dict{Symbol, PSI.ServiceModel}()
+devices = Dict{Symbol, DeviceModel}(:Generators => DeviceModel(PSY.ThermalStandard, PSI.ThermalBasicUnitCommitment),
+                                    :Ren => DeviceModel(PSY.RenewableDispatch, PSI.RenewableFixed),
+                                    :Loads =>  DeviceModel(PSY.PowerLoad, PSI.StaticPowerLoad),
+                                    :ILoads =>  DeviceModel(PSY.InterruptibleLoad, PSI.StaticPowerLoad),
+                                    )       
+template_uc= OperationsTemplate(CopperPlatePowerModel, devices, branches, services);
+
+## ED Model Ref
+branches = Dict{Symbol, DeviceModel}()
+services = Dict{Symbol, PSI.ServiceModel}()
+devices = Dict{Symbol, DeviceModel}(:Generators => DeviceModel(PSY.ThermalStandard, PSI.ThermalDispatchNoMin, SemiContinuousFF(:P, :ON)),
+                                    :Ren => DeviceModel(PSY.RenewableDispatch, PSI.RenewableFullDispatch),
+                                    :Loads =>  DeviceModel(PSY.PowerLoad, PSI.StaticPowerLoad),
+                                    :ILoads =>  DeviceModel(PSY.InterruptibleLoad, PSI.DispatchablePowerLoad),
+                                    )       
+template_ed= OperationsTemplate(CopperPlatePowerModel, devices, branches, services);
+
+GLPK_optimizer = with_optimizer(GLPK.Optimizer)
+
+stages = Dict(1 => Stage(template_uc, 24, Hour(24), 1, c_sys5_uc, GLPK_optimizer,  Dict(0 => Sequential())),
+              2 => Stage(template_ed, 12, Minute(5), 24, c_sys5_ed, GLPK_optimizer, Dict(1 => Synchronize(24,1), 0 => Sequential()), TimeStatusChange(:ON_ThermalStandard)))
+
+sim = Simulation("test", 2, stages, "/Users/lhanig/Documents/"; verbose = true)
```

Comment:
You can't specify a directory that is local to you, try something inside the package like this 
`test_dir = joinpath(base_dir,"test/results"); mkdir(test_dir)`